chore: added vitty to shortened URLs

VisheshBansal opened the pull request:
Added two different redirects for vitty extension as playstore has vitty-extn whereas the app has vitty-extension.

@BRO3886 Pls review and merge

BRO3886 commented:
I updated the handler so now https://dscv.it/app/app-package-name will automatically handle any play store app:

https://dscv.it/app/vitty

@VisheshBansal I think you can remove app redirect, we can leave it for a bit if it's hard-coded to the extension.

As for the extension, I updated the listing so now it's vitty-extension on play store

VisheshBansal commented:
> I updated the handler so now https://dscv.it/app/app-package-name will automatically handle any play store app:
> 
> https://dscv.it/app/vitty
> 
> @VisheshBansal I think you can remove app redirect, we can leave it for a bit if it's hard-coded to the extension.
> 
> As for the extension, I updated the listing so now it's vitty-extension on play store

I think it is hardcoded into the extension, so that's why I have kept it for now. If it changes, I will remove the redirect. As for the extension, I'll just update it

VisheshBansal commented:
Done, removed the duplicate. @BRO3886 pls review and merge
